### PR TITLE
TD(lambda) fix

### PR DESCRIPTION
**Pull request by taodav**
Only the TD(lambda) fix, as well as some refactoring to clean up relative imports.

All the scripts still work, and all tests seem to be passing.

Make sure to do a `pip install -e .` after this merge!

### CODE REVIEW

**Comment by dsctt** [comment]
Try `yapf --recursive --in-place .` to fix the formatting test

**Comment by taodav** [comment]
@dsctt I'm running on Python 3.8.10 and getting an error similar to [this](https://github.com/google/yapf/issues/403). Not sure exactly what character is causing it.

**Comment by camall3n** [comment]
We do have some unicode symbols, but the file that needed formatting was `test_td_lambda.py`, which AFAIK doesn't have weird symbols in it at all. What error does yapf give you when you run it?

**Comment by camall3n** [comment]
Meanwhile, the build still [fails](https://github.com/camall3n/grl/actions/runs/3124150368/jobs/5067253589), because it's confused about the imports

```
==================================== ERRORS ====================================
__________________ ERROR collecting tests/test_get_policy.py ___________________
ImportError while importing test module '/home/runner/work/grl/grl/tests/test_get_policy.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
/opt/hostedtoolcache/Python/3.10.6/x64/lib/python3.10/importlib/__init__.py:126: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
tests/test_get_policy.py:3: in <module>
    from grl.grl.mdp import MDP
E   ModuleNotFoundError: No module named 'grl.grl'
=========================== short test summary info ============================
ERROR tests/test_get_policy.py
!!!!!!!!!!!!!!!!!!!! Interrupted: 1 error during collection !!!!!!!!!!!!!!!!!!!!
=============================== 1 error in 1.29s ===============================
Error: Process completed with exit code 2.
```

**Comment by taodav** [comment]
Fixed the imports, `yapf`ed all files and committed those changes.

**Review by dsctt — APPROVED** [review]
lgtm

File: grl/mdp.py [hunk 1]
```diff
@@ -289,4 +287,4 @@ def test():
     print('All tests passed.')
 
 if __name__ == '__main__':
-    test()
```
Comment (camall3n) [inline grl/mdp.py:292]: Is there a way to get yapf to take care of ensuring there's a newline at the end of every file?

**Review by camall3n — APPROVED** [review]
I'm happy with this if you add the missing end-of-file newline.

File: grl/mdp.py (hunk 1 above)
Comment (camall3n) [reply grl/mdp.py:292]: Maybe we can use this: https://github.com/editorconfig-checker/editorconfig-checker/

Comment (camall3n) [reply grl/mdp.py:292]: Actually this is probably way easier:

https://gist.github.com/OleksiyRudenko/d51388345ea55767b7672307fe35adf3

**Pull request merged**